test/support/ssl/insecure-cert.pem

```diff
@@ -0,0 +1,34 @@
+-----BEGIN CERTIFICATE-----
+MIIF9DCCA9ygAwIBAgIJAJhLP5JrvONkMA0GCSqGSIb3DQEBBQUAMFkxCzAJBgNV
```

Comment:
I started looking for a package with a built in certificate but the first few I saw looked off the mark. I may have given up early on the search, dunno.